config/initializers/pulmap_config.rb

```diff
@@ -0,0 +1,13 @@
+module Pulmap
+  def config
+    @config ||= config_yaml.with_indifferent_access
+  end
+
+  private
+
+    def config_yaml
+      YAML.load(ERB.new(File.read("#{Rails.root}/config/config.yml")).result)[Rails.env]
```

Comment:
could the config.yml file be renamed to be more specific? there are a lot of ymls in the config directory